src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,75 @@
-const Countdown = () => null;
+import { useState, useEffect } from "react";
 
-export default Countdown;
+
+const StopWatch = () => {
+// set the initial time to 2.5 minutes
+  const [time, setTime] = useState<number>(60000 * 2.5);
+  // set the initial state of the stopwatch to not running
+  const [isRunning, setIsRunning] = useState<boolean>(false);
+
+  useEffect(() => {
+    let interval: number;
+      // check if the stopwatch is running and the time is greater than 0
+    if (isRunning && time >= 0) {
+      interval = setInterval(() => {
```

Comment:
setInterval returns an interval that repeats every X ms. If you are going to clear the interval and recreate it each time, there's no point in using an interval but rather setTimeout, which runs just once.